static-assets/scripts/host.js

```diff
@@ -609,17 +609,21 @@
     }
 
     if (siteChanged || !hasCheckIn) {
-          win.src = previewAppBaseUri + hash.page;
+      win.src = previewAppBaseUri + hash.page;
     } else {
       communicator.publish(Topics.CHANGE_GUEST_REQUEST, {
         base: previewAppBaseUri,
         url: hash.page
       });
-      }
+    }
 
     var path = hash.page,
       hashPage = hash.page;
 
+    if (path.startsWith('/static-assets/')) {
```

Comment:
instead of startsWith, match against `^((\/static-assets)|(\/remote-assets)|(\/api))`